gensim/models/wrappers/fasttext.py

```diff
@@ -348,6 +361,16 @@ def init_ngrams(self):
             self.wv.ngrams[ngram] = i
         self.wv.syn0_all = self.wv.syn0_all.take(ngram_indices, axis=0)
 
+        ngram_weights = self.wv.syn0_all
+
+        for w, v in self.wv.vocab.items():
+            word_ngrams = self.compute_ngrams(w, self.wv.min_n, self.wv.max_n)
+            for word_ngram in word_ngrams:
+                self.wv.syn0[self.wv.vocab[w].index] += np.array(ngram_weights[self.wv.ngrams[word_ngram]])
+
+            self.wv.syn0[self.wv.vocab[w].index] /= (len(word_ngrams) + 1)
+        logger.info("loaded %s matrix from %s" % (self.wv.syn0.shape))
```

Comment:
Okay, but you're passing in a tuple, so the message is going to look something like `loaded 30000 matrix from 100`. Please update the message.